sdk/core/Azure.Core.Experimental/src/Primitives/BinaryData.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+namespace Azure.Core.Experimental.Primitives
```

Comment:
Why `.Primitives`?  I would expect this lives in just `Azure.Core` or maybe even `Azure` someday.